edx_proctoring/api.py

```diff
@@ -493,6 +493,26 @@ def remove_allowance_for_user(exam_id, user_id, key):
         emit_event(exam, 'allowance.deleted', override_data=data)
 
 
+def add_bulk_allowances(exam_ids, user_ids, allowance_type, value):
+    """
+    Adds (or updates) an allowance for multiple users and exams
+    """
+    exam_ids = set(exam_ids)
+    user_ids = set(user_ids)
+    if allowance_type == 'time_multiplier':
```

Comment:
Instead of string, we should define these options as python ENUMs. See the `constants.py` for examples.